src/pcapi/connectors/api_recaptcha.py

```diff
@@ -20,7 +19,7 @@ class ReCaptchaException(Exception):
     pass
 
 
-def validate_recaptcha_token(token: str, original_action: str) -> bool:
+def validate_recaptcha_token(token: str, original_action: str, score: float) -> bool:
```

Comment:
```suggestion
def validate_recaptcha_token(token: str, original_action: str, required_score: float) -> bool:
```

... ou bien `minimal_score` ?

Reply:
Allez :+1: